frontend/src/main.ts

```diff
@@ -60,8 +60,9 @@ async function getFirebaseConfig(): Promise<Object> {
 
 getFirebaseConfig().then(config => {
   firebase.initializeApp(config);
-  firebase.auth().onAuthStateChanged(user => {
-    store.dispatch("user/handleUser", user);
+  firebase.auth().onAuthStateChanged(async user => {
+    await store.dispatch("user/handleUser", user);
+    store.dispatch("user/handleMetaData");
```

Comment:
Why can't we move this code altogether into the `user` store module? I mean hooking on `onAuthStateChanged`. From here we can call an action of the `user` module indicating that the config is obtained, and that the module can use Firebase functionality. It may be even more general store event.